mod-source-record-manager-server/src/main/java/org/folio/services/ParsedRecordServiceImpl.java

```diff
@@ -30,6 +15,19 @@
 import org.folio.rest.jaxrs.model.SourceRecord;
 import org.folio.rest.jaxrs.model.SourceRecordState;
 import org.folio.services.mappers.processor.MappingParametersProvider;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.ws.rs.NotFoundException;
+import java.util.HashMap;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static io.vertx.core.Future.failedFuture;
+import static java.lang.String.format;
+import static org.folio.HttpStatus.HTTP_NOT_FOUND;
+import static org.folio.HttpStatus.HTTP_OK;
+import static org.folio.verticle.consumers.util.QMEventTypes.QM_RECORD_UPDATED;
```

Comment:
Your import is not in proper order.

* import static all other
* empty line
* import static org.folio.*
* empty line
* import java/javax.*
* empty line
* import all other
* empty line
* import org.folio.*